### DISCUSSION TITLE
Is there a method to implement a responsive grid layout

### DISCUSSION THREAD

**Discussion by shaneholloman** [squidfunk/mkdocs-material#2848] [Ideas]
Gatsby has a method to responsively divide the page like so:  
<https://gatsby-remark-grid-system.netlify.app/>
Plugin Used: <https://www.gatsbyjs.com/plugins/@pauliescanlon/gatsby-remark-grid-system/>

Material uses this:  
<https://material.io/design/layout/responsive-layout-grid.html#columns-gutters-and-margins

Has anyone, managed to implement something like this yet?

```markdown
::: div row
    ::: div col 12|6|7|10
        left content
    :::
    ::: div col 12|6|5|2
        right content
    :::
:::
```

**Comment by squidfunk** [comment]
There’s no Markdown extension that I know of, but you can use plain HTML with some CSS to achieve what you want.
